apps/nextjs-website/src/messages/it.json

```diff
@@ -48,6 +48,22 @@
                 "email": "Indirizzo e-mail",
                 "password": "Password"
             }
+        },
+        "changePassword": {
```

Comment:
There's this error message to fix:
![image](https://github.com/pagopa/developer-portal/assets/39835990/8d2488b9-636d-48cb-85e5-cd7123b0091a)

![image](https://github.com/pagopa/developer-portal/assets/39835990/4e21b2ff-f66e-4ba0-882c-1b8088008ed2)

Reply:
we need at least one special character, so i think that design must be updated

Reply:
Does the sentence wrap?